### DISCUSSION TITLE
Suggestion - Octoprint VM or LXC

### DISCUSSION THREAD

**Discussion by WWE-Corey** [tteck/Proxmox#430] [Ideas]
Given that Octoprint is a pretty popular and useful printer manager/interface it might be a worthwhile project to create. As powerful as it is, it is designed to be a single instance- so if you had a helper script, a person could easily deploy multiple instances to manage multiple printers.

**Comment by smarthomelawyer** [comment]
Agreed!

**Reply by bam27beer** [reply]
tried on docker an unbunto server ,couple of you tube vids ,(chris's basement is one of them )
but unless i do it direct to a pi ,i can't work it out ,
mind i'm very green on linux 
would love to run my e3v2 and 5pro from my micro machine ,
the i can get rid of the 4b 8gb and 3b pi's

**Reply by Vakrehus** [reply]
LXC with 2 cores, 2 GB RAM, 32GB disk size on debian buster running octoprint. pass trough usb for the printer and you're done.

**Reply by WWE-Corey** [reply]
I ended up doing the in ProxMox for 3 instances. Worked out well.

**Reply by bam27beer** [reply]
ha after months ,kinda followed your route , not completly installed yet as waiting for 2 usb cameras and a hub to turn up ,was using pi cams, hp G2 260 mini ,so not too many usb ports ,
https://www.youtube.com/watch?v=J5VzI4AFav4&lc=
